apps/cli/templates/vercel/package.json

```diff
@@ -0,0 +1,6 @@
+{
+  "dependencies": {
+    "@remix-run/vercel": "^1.17.1",
+    "@vercel/node": "^2.10.3"
```

Comment:
Tricky one. Vercel now maintains own `@vercel/remix` as a replacement for these two. `@remix-run/vercel` is gonna be deprecated.

Reply:
So, should this template adopt the `@vercel/remix` package then ?

Reply:
Remix says we may not need remix template at all. Like vercel can install own stuff when remix is detected.

https://github.com/remix-run/remix/tree/main/templates/vercel

Reply:
ok, dropping vercel template then 👍